charts/camunda-platform-alpha/test/unit/camunda/documentstore_configmap_test.go

```diff
@@ -1,31 +1,14 @@
-// documentstore_configmap_test.go
-// Copyright 2022 Camunda Services GmbH
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
Was it intentional to remove the license? I wonder if part of this file was copied from elsewhere since we have it.

Reply:
Oh thanks - was not intentional 🙏